components/bounty/bounty-hero/bounty-settings/view.tsx

```diff
@@ -0,0 +1,241 @@
+    }) {
+    return (
+        <a
+          href={`https://github.com/${forcePath}/${hrefPath}`}
+          target="_blank"
+          rel="noreferrer"
+          className="text-decoration-none text-white hover-gray"
+          onClick={onClick}
+        >
+          {children} <ArrowUpRight className="ms-1" />
+        </a>
+    );
+  }
+  
+  function renderViewPullRequestLink() {
+    if (
+        isWalletConnected &&
+        isGithubConnected &&
+        !isBountyInDraft &&
+        hasOpenPullRequest
+      )
+      return (
+          <span className="d-flex py-2 cursor-pointer">
+            <GithubLink
+              forcePath={bounty?.repository?.githubPath}
+              hrefPath={`pull?q=base:${bounty?.branch}`}
+            >
+              <Translation ns="pull-request" label="actions.view" />
+            </GithubLink>
+          </span>
+      );
+  }
+  
+  function renderEditButton() {
+    if (isWalletConnected && isBountyInDraft && isBountyOwner)
+      return (
+          <span className="py-2 cursor-pointer" onClick={handleEditIssue}>
+            <Translation ns="bounty" label="actions.edit-bounty" />
+          </span>
+      );
+  }
+  
+  function renderCancel() {
+    const Cancel = (isHard: boolean) => (
+        <span
+          className="py-2 cursor-pointer"
+          onClick={() => (isHard ? setShowHardCancelModal(true) : handleRedeem())}
+        >
+          <Translation
+            ns={isHard ? "common" : "bounty"}
+            label={isHard ? "actions.cancel" : "actions.owner-cancel"}
+          />
+        </span>
+      );
+  
+    if (network?.active?.isGovernor && isCancelable)
+      return Cancel(true);
+  
+    const isDraftOrNotFunded = isFundingRequest
+        ? !isBountyFunded
+        : isBountyInDraft;
+  
+    if (
+        isWalletConnected &&
+        isBountyOpen &&
+        isBountyOwner &&
+        isDraftOrNotFunded &&
+        !isEditIssue
+      )
+      return Cancel(false);
+  }
+  
+  function renderActions() {
+    return (
+        <>
+          <span className="d-flex pb-2 cursor-pointer">
+            <GithubLink
+              forcePath={bounty?.repository?.githubPath}
+              hrefPath={`${
+                (bounty?.state?.toLowerCase() ===
+                  "pull request" &&
+                  "pull") ||
+                "issues"
+              }/${bounty?.githubId || ""}`}
+              onClick={
+                !network?.repos?.active?.ghVisibility
+                  ? () => setShowGHModal(true)
+                  : null
+              }
+            >
+              {t("actions.view-on-github")}
+            </GithubLink>
+          </span>
+          {renderViewPullRequestLink()}
+          {renderEditButton()}
+          {renderCancel()}
+        </>
+    );
+  }
+  
+  useEffect(loadOutsideClick, [show]);
+  
+  return (
+      <>
+        <div className="position-relative d-flex justify-content-end" ref={node}>
+          <div
```

Comment:
why don't we use a normal dropdown like we don on the rest of the app ?

Reply:
I followed the same logic that was built in the filter component. Because I needed a customization that react-select doesn't provide.

`components\issue-filters.tsx`